src/components/Avatar/__test__/Avatar.test.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import Avatar from "../"
+import { render, cleanup, fireEvent } from "react-testing-library"
+import { ThemeProvider } from "../../../common/styles"
+import "jest-styled-components"
+import "jest-dom/extend-expect"
+
+afterEach(cleanup)
+
+const defaultProps = {
+  size: "24px",
+  disabled: false,
+  onClick: () => {},
+}
+
+const additionalProps = {
+  picture: "https://i.ibb.co/DwVLSqF/user-profile.jpg",
+  name: "Jane Doe",
+}
+
+describe("Avatar Snapshots", () => {
+  it("Picture avatar", () => {
+    const { asFragment } = render(
+      <Avatar {...defaultProps} {...additionalProps} />,
+      { wrapper: ThemeProvider }
+    )
+    expect(asFragment()).toMatchSnapshot()
+  })
+  it("Name avatar", () => {
+    const { asFragment } = render(
+      <Avatar {...defaultProps} name={additionalProps.name} />,
```

Comment:
You can also put an explicit assertion for the first letter that should be displayed in the avatar.